### PR TITLE
Bump CNAO to 0.39.2

### PR DESCRIPTION
**Pull request by qinqon**
It includes a lot of fixes from 0.35

Signed-off-by: Quique Llorente <ellorent@redhat.com>

### CODE REVIEW

File: cluster-provision/k8s/1.18/manifests/cnao/network-addons-config.crd.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ spec:
           type: object
         status:
           type: object
+      type: object
```
Comment (maiqueb) [inline cluster-provision/k8s/1.18/manifests/cnao/network-addons-config.crd.yaml:29]: 0_o ... why is this suddenly needed ?

Comment (qinqon) [reply cluster-provision/k8s/1.18/manifests/cnao/network-addons-config.crd.yaml:29]: No clue, I downloaded from CNAO release pagehttps://github.com/kubevirt/cluster-network-addons-operator/releases/tag/0.39.2

Comment (phoracek) [reply cluster-provision/k8s/1.18/manifests/cnao/network-addons-config.crd.yaml:29]: It became a requirement for CRDs in recent versions. Previously, the top level object was treated as `object` implicitly, but it was added that it has to be explicit.

**Comment by phoracek** [comment]
/hold

We need to update image hashes

**Comment by phoracek** [comment]
@qinqon would you update hashes?

**Comment by qinqon** [comment]
/hold cancel
shas updated.

**Pull request merged**